src/PowerShellEditorServices/Services/PowerShellContext/Session/PSReadLinePromptContext.cs

```diff
@@ -98,7 +80,9 @@ internal class PSReadLinePromptContext : IPromptContext
             {
                 pwsh.Runspace = runspace;
                 var psReadLineType = pwsh
-                    .AddScript(ReadLineInitScript, useLocalScope: true)
+                    .AddCommand("Import-Module")
+                        .AddParameter("Name", _psReadLineModulePath.Replace("'", "''"))
```

Comment:
I wouldn't have thought the escaping here was needed, since it's already being defined as the value associated with the parameter, although admittedly I don't know for sure.